helm/azure/cert-fetch/templates/issuer.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.managedCertificate }}
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: letsencrypt
+  namespace: istio-system
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
```

Comment:
what is acme?

Reply:
This is the LetEncrypt certificate provider. It automatically issues a certificate free of cost.